### DISCUSSION TITLE
Sort by last modified

### DISCUSSION THREAD

**Discussion by m3lting** [deckerst/aves#658] [Ideas]
I would love to be able to sort by last modified just like it is possible in the base samsung gallery as well on windows. Hope its not tooooo difficult to add so you could do that. Besides that, best app ever with everything needed ^^

**Comment by deckerst** [comment]
Actually, there is already an issue requesting this:
- #97

I explain in that issue why I haven't implemented it so far. If you have interesting use cases, feel free to comment on that issue.
